shared/js/background/atb.js

```diff
@@ -175,6 +243,11 @@ const ATB = (() => {
                         return !!atb
                     })
 
+                    // in case there is no assigned atb, enroll into Chrome Counteract experiment
+                    if (!atb) {
+                        atb = ATB.getChromeCounteractExpATB(ATB.getATBVersionInfo().version)
```

Comment:
I don't think we need the `getATBVersionInfo` implementation here, as `ATB.setInitialVersions` will already have pulled a plain ATB version from the server which you can get with `settings.getSetting('atb')`. Then it's just a case of appending the experiment.